### PR TITLE
Database integration

### LINKED ISSUE #9: A user can upload a picture along with a caption for other users to see
User Story:
User can select a photo from their library and write a caption that will be included right under the picture for followers to see.

### PR DESCRIPTION
**Pull request by MatteoGisondi**
close #9 #10 #11 #85 #84

Significant progress towards database integration. Any subsequent changes should be completed in a specialized branch.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -107,98 +96,150 @@ def logout():
...
+    form = ResetPasswordForm()
+    if request.method == "POST":
+        if form.validate_on_submit():
+            # gets information of password from form
+            password = form.password.data
+            password2 = form.password2.data
+            # checks to see if password is good
+            if password != password2:
+                return redirect("/resetPassword/"+emailadress)
+            user = User.get_by_email(db, emailadress)
```
Comment (alexbara2000) [inline app.py]: I get an error here whenever running the app, it doesnt recognize "User"

**Review by alexbara2000 — APPROVED** [review]

**Review by steven-arsan — APPROVED** [review]
Looks good on my end.

File: user.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    # print(User.get_usernames(db))
+    user = User.get_by_username(db, 'Ablion73')
+    # print(user.get_user(dictionary=True, hide_password=True))
+    user_by_id = User.get_by_id(db, 1)
+    assert user == user_by_id, 'ids are different'
+    # followable = user.get_followable(db)
+    # print(user.id)
+    # print(followable)
+    # print(user.get_likes(db))
+    print(user.get_following_post_images(db))
```
Comment (LiamBurchellReyes) [inline user.py]: Are these supposed to not be commented (lines 228 - 236)? Are they for testing or for regular functionality?

**Review by LiamBurchellReyes — APPROVED** [review]
Looks good to me!

**Review by jongjonathan — APPROVED** [review]
After installing mysql.connector and going through the features, it works very well on my end! Able to unfollow, follow, see who liked posts... Great work on database and feature changes!

**Comment by JosephMezza** [comment]
Installed mysql.connector and all the features of the website work great! Good job guys

**Review by JosephMezza — APPROVED** [review]
Installed mysql.connector and all the features of the website work great! Amazing job guys

**Pull request merged**